tanium-threat-response/src/openbas_tanium_threat_response.py

```diff
@@ -180,6 +178,12 @@ def _match_alert(self, endpoint, alert, expectation):
                     "data": self._extract_process_names(alert_details),
                     "score": 80,
                 }
+            if type == "parent_process_name":
+                alert_data[type] = {
```

Comment:
Suggestion: if type in ["process_name", "parent_process_name"]:

Reply:
Good catch, I fixed it. Also, I increased the matching threshold because we want it to be exact (leaving 5% margin to match with/without ".exe")